boards/opentitan/src/main.rs

```diff
@@ -209,6 +212,10 @@ pub unsafe fn reset_handler() {
         FAULT_RESPONSE,
         &process_mgmt_cap,
     );
-
+    let bad_ptr = 0x81200000 as *mut u8;
+    let val = *bad_ptr;
+    if val == 0 {
+        debug!("Dereferenced bad pointer.");
+    }
```

Comment:
Probably want to remove this?

Reply:
Yes, before merging. But having this lets you test what a printout looks like. :)

Reply:
Make sure this is dropped.